src/lib.rs

```diff
@@ -344,6 +398,7 @@ pub struct AtlasSprite3d {
     /// An emissive colour, if the sprite should emit light.
     /// `Color::Black` (default) does nothing.
     pub emissive: Color,
+    pub base_color: Color,
```

Comment:
It would be the same story as line 204/249 here, but if you pull from main into your PR branch this whole struct actually goes away in the newer version.